src/openrct2/actions/FootpathPlaceAction.cpp

```diff
@@ -277,8 +277,9 @@ GameActions::Result::Ptr FootpathPlaceAction::ElementInsertQuery(GameActions::Re
 GameActions::Result::Ptr FootpathPlaceAction::ElementInsertExecute(GameActions::Result::Ptr res) const
 {
     bool entrancePath = false, entranceIsSamePath = false;
+    uint32_t flags = GetFlags();
 
-    if (!(GetFlags() & (GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_GHOST)))
+    if (!(flags & (GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_GHOST)))
```

Comment:
don't make unrelated changes in this PR save it for a different PR.

Reply:
wait... how did this end up here? dang I think I must have been working on the wrong local branch when I was investigating something. I'll revert this on my next commit to this PR